infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoController.kt

```diff
@@ -45,7 +52,7 @@ class RatkoController(
     @PostMapping("/push-location-tracks")
     fun pushLocationTracksToRatko(@RequestBody changes: List<LocationTrackChange>): ResponseEntity<String> {
         logger.apiCall("pushLocationTracksToRatko", "changes" to changes)
-        ratkoService.pushLocationTracksToRatko(changes)
+        ratkoService.pushLocationTracksToRatko(LayoutBranch.main, changes)
```

Comment:
Manuaalisessa pushissa ratkoonkin tarvitaan branch. En laittanut sitä vielä API:in koska ratkoviennit pitää tarkastella joka tapauksessa vielä tarkemmin. Laition sen kuitenkin tästä kohtaa service-kutsuun, jotta voi nätimmin korjata data-haut käyttämään uusia metodeita.